app/test_main.py

```diff
@@ -1,3 +1,21 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+class Test:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, result",
+        [
+            (28, 28, [3, 2]),
```

Comment:
Please add more tests for different boundary values, such as (14, 14), (27, 27) ...